### PR TITLE
First page for mobile

### PR DESCRIPTION
**Pull request by SulimanJoya**
What did in this project 
1. I create a repo on GitHub (My-Portfolio)
2. clone to my local machine by (git clone) command 
3. create a new branch for my work flow
4. setup the linters for my project 
5. Add .gitignore file 
6. Add a README file 
7. Add index.html file
8. Add styles.css file
9. create a pull request

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #bbb7b7;
+}
```
Comment (mukhammadrizooff) [inline styles.css]: - [ ] [REQUIRED] Good job so far! The `background-color` value given to `Body` is reflected on the `right` side in the mobile version, please fix it 😃 

image

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+  I set up my portfolio repository. My goal here was to set up and customize my portfolio for mobile and desktop versions.
+
+- ![screenshot](/images/project-screenshot.JPG)
+
+## :hammer: Built With
+
+- HTML
+- CSS
+
+## Live Demo
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] If there is no `Live demo` link for your project, please remove this text(`## Live Demo`) from the `README` file or write down the information that it is not available 😃

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @SulimanJoya, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive Commit history ✔️
- Descriptive `PR` description ✔️  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @SulimanJoya , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**